mbid_mapping/mapping/mb_release_group_cache.py

```diff
@@ -133,9 +133,16 @@ def create_json_data(self, row):
                 tag["genre_mbid"] = genre_mbid
             release_group_tags.append(tag)
 
+
+        date = str(row["year"] or '')
+        if row["month"] is not None:
```

Comment:
Is this the same format that MB follows for partial dates because consider a date where the month is unknown but the year and day are, it could be formatted as 2024-01 which is easily misunderstood.

Reply:
MB dates are not allowed to have a day without a month. YYYY, YYYY-MM, YYYY-MM-DD are the only valid forms.

Reply:
Ah cool, awesome!